CefSharp.BrowserSubprocess/CefSharp.BrowserSubprocess.csproj

```diff
@@ -110,8 +110,11 @@
   <PropertyGroup>
     <PostBuildEvent>if $(PlatformName) == x86 (
   call "$(DevEnvDir)..\tools\vsvars32.bat"
-  editbin /largeaddressaware $(TargetPath)
-)</PostBuildEvent>
+  editbin /largeaddressaware $(TargetPath)  /TSAWARE $(TargetPath)
```

Comment:
I think it is not necessary
https://msdn.microsoft.com/en-us/library/8e77sz74.aspx